### ISSUE TITLE
Switch to Axios

### ISSUE THREAD

**Issue by andrevandal** [RealFaviconGenerator/rfg-api#14]
hi, i get there message in npm audit

```
  Low             Regular Expression Denial of Service
  Package         debug
  Patched in      >= 2.6.9 < 3.0.0 || >= 3.1.0
  Dependency of   nuxt-rfg-icon [dev]
  Path            nuxt-rfg-icon > rfg-api > node-rest-client > debug
  More info       https://nodesecurity.io/advisories/534
```

[same issue on nuxt-rfg-icon](https://github.com/pimlie/nuxt-rfg-icon/issues/8#issuecomment-425491059)

**Comment by phbernard** [comment]
Apparently the problem is from [node-rest-client](https://github.com/aacerox/node-rest-client/pull/164).

The best alternative would be to twitch to another rest lib, such as Axios.

**Comment by phbernard** [comment]
Implemented and pushed. The package will be deployed in a few days.

**Comment by phbernard** [comment]
Published a minute as 0.4.0.
